components/SeoHead.js

```diff
@@ -0,0 +1,89 @@
+import NextHead from 'next/head'
+import { Component } from 'react'
+
+const defaultTitle = 'styled-components'
```

Comment:
Can you use `defaultProps` instead of the OR behaviour below? Should clean things up a little